apps/reports/src/components/ListReports.vue

```diff
@@ -32,7 +32,7 @@
       >
       <TableSimple
         @rowClick="open"
-        :columns="['id', 'name']"
+        :columns="['index', 'id', 'name']"
```

Comment:
we could choose to stop the 'name'

Reply:
Disagree, unless we also include 'description'. We'd want to keep the ids concise, but without 'name' or 'description' it would be hard to tell what the query is for

Reply:
I agree. I would then propose to rename it to something more informative. Would you like both a 'label' and a 'description'?